fpm.toml

```
@@ -1,6 +1,9 @@
-name = "functional-fortran"
+name = "functional"
 version = "0.5.0"
 license = "BSD-3-Clause"
+author = "Milan Curcic"
+maintainer = "mcurcic@wavebitscientific.com"
+copyright = "Copyright (c) 2016-2020, Milan Curcic"
```

```suggestion
copyright = "Copyright (c) 2016-2020, functional-fortran contributors"
```

Unless you don't accept the copyright being the only other contributor :)

---

Not really a fan of the author/copyright/maintainer entries in the package manifest, see https://github.com/fortran-lang/fpm-registry/issues/24. Makes it even harder to keep all the meta data in the repo consistent.

In this case the fpm.toml would mismatch the header in the BSD-3-License:
https://github.com/wavebitscientific/functional-fortran/blob/f9baf451a3256c84165c2b9a3fdffd335ba1124e/LICENSE#L1

---

Yes, the LICENSE file should as well be updated in this PR.

I tend to agree, let's discuss it there. Here, feel free to remove or keep any of these optional fields. I don't think they're essential, nor do they bother me.